src/sonic-config-engine/sonic-cfggen

```diff
@@ -41,9 +41,10 @@ from minigraph import minigraph_encoder
 from minigraph import parse_xml
 from minigraph import parse_device_desc_xml
 from minigraph import parse_asic_sub_role
-from portconfig import get_port_config, get_port_config_file_name, get_breakout_mode
+from portconfig import get_port_config, get_breakout_mode
 from sonic_py_common.device_info import get_platform, get_system_mac
 from sonic_py_common.multi_asic import get_asic_id_from_name, is_multi_asic
+from sonic_py_common import device_info
```

Comment:
Two functions from this module are already imported on line 45. Suggest replacing line 45 with this line and adding `device_info.` prefix to all references to `get_platform()` and `get_system_mac()` in this file.

Reply:
fixed